SadConsole/Readers/Playscii.cs

```diff
@@ -201,9 +195,12 @@ public ColoredGlyph ToColoredGlyph(IFont font, Palette colors)
         static Palette GetPalette(string fileName)
         {
             if (!File.Exists(fileName)) throw new FileNotFoundException("Palette file doesn't exist.");
+            string ext = Path.GetExtension(fileName).ToLower();
+            if (!s_paletteExtensions.Contains(ext))
+                throw new InvalidOperationException("Palette file extension not supported by the Playscii format.");
 
-            // check if this palette has already been created previously
-            string paletteName = Path.GetFileName(fileName);
+                // check if this palette has already been created previously
+                string paletteName = Path.GetFileName(fileName);
```

Comment:
I think the indentation is wrong here
```suggestion
            // check if this palette has already been created previously
            string paletteName = Path.GetFileName(fileName);
```